Problem4/src/problem4/Ball.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem4;
+import java.util.Random;
+/**
+ * Bouncing Ball object, used to store each ball's speed, size, and location
+ * 
+ * @author Evan
+ */
+public class Ball
```

Comment:
Your helper Ball class should be implemented using OOP best practices.  Member variables should be private with appropriate getters and setters.  You should also be using a constructor to initialize the ball’s radius, positioning and movement speed.  This would have allowed you to easily vary the colour and radius of each ball (if you wanted).